app/src/main/java/com/example/parkingsystemjava/mvp/presenter/DateReservationDialogPresenter.java

```diff
@@ -0,0 +1,7 @@
+package com.example.parkingsystemjava.mvp.presenter;
+
+import com.example.parkingsystemjava.mvp.contract.ParkingReservationDialogContract;
+
+public class DateReservationDialogPresenter implements ParkingReservationDialogContract.ReservationActivityPresenter {
```

Comment:
if you are not using this now, it should not be here. Please remove it

Reply:
done